prowler/lib/outputs/json.py

```diff
@@ -78,6 +78,9 @@ def fill_json_asff(finding_output, audit_info, finding, output_options):
         AssociatedStandards=associated_standards,
         RelatedRequirements=compliance_summary,
     )
+    # Fill Recommendation Url if it is blank
+    if not finding.check_metadata.Remediation.Recommendation.Url:
```

Comment:
Could you create a test for this case please?

Reply:
We cannot create it since moto does not cocer this call and we cannot mock the call twice.

Reply:
This has nothing to do with Moto, is just a test for the `fill_asff_json`. I'm finishing it and I'll push it later today.

Reply:
Thanks 😄